src/rest_api.py

```diff
@@ -44,6 +53,55 @@ def liveness():
     return flask.jsonify({}), 200
 
 
+@app.route('/api/v1/sync-graph-data', methods=['POST'])
+@login_required
+def sync_data():
+    """
+    Endpoint for carrying a sync of graph db data.
+
+    Takes in value to either call sync of non cve version
+    or latest version or both.
+
+    valid input: {
+        "non_cve_sync": True / False,
+        "latest_version_sync": True / false,
+        "cve_ecosystem": ['maven', 'pypi, 'npm']
+    }
+
+    """
+    resp = {
+        "success": True,
+        "message": "Sync operation started ->"
+    }
+
+    input_json = request.get_json()
+    logger.info("sync-graph-data called with the input {i}".format(i=input_json))
+    non_cve_sync = input_json.get('non_cve_sync', False)
+    if non_cve_sync == "true":
```

Comment:
@yzainee  as per the swagger spec, `'non_cve_sync'` is a boolean field and should not be compared with a string `"true"`.

The boolean value comparison in Python happens like `if non_cve_sync:` or like `if not non_cve_sync`.